src/material-examples/sticky-header-overview/sticky-header-overview-example.css

```diff
@@ -0,0 +1,75 @@
+.sticky-header-example .example-centered {
+  text-align: left;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  height: 300px;
+  width: 500px;
+  overflow: auto;
+}
+
+.sticky-header-example button {
+    position: fixed;
+    margin: 16px;
+    z-index: 99;
+}
+
+.sticky-header-example * {
```

Comment:
Is there a way to make this more specific than `*`? The wildcard is generally discouraged